app/services/location_data_service.rb

```diff
@@ -146,14 +146,15 @@ def holding_library_id(holding_library_code)
       library.id if library.present?
     end
 
+    # adds firestone$ssrcfo and firestone$ssrcdc with requestable: true
     def update_holding_location_record
       location_code = ["firestone$ssrcfo", "firestone$ssrcdc"]
       location_code.each do |lc|
         holding_location_record = HoldingLocation.find_by(code: lc)
-        holding_location_record.open = "true"
-        holding_location_record.requestable = "false"
-        holding_location_record.always_requestable = "false"
-        holding_location_record.save
+        if holding_location_record.present?
```

Comment:
What does this block of code do?

Reply:
@mzelesky it sets the specific location to have requestable:'false'. The default requestable value from locations is 'true'.
 I will update the comment as you requested in slack.